src/Compiler/FSComp.txt

```diff
@@ -1641,3 +1641,4 @@ reprStateMachineInvalidForm,"The state machine has an unexpected form"
 3519,tcInlineIfLambdaUsedOnNonInlineFunctionOrMethod,"The 'InlineIfLambda' attribute may only be used on parameters of inlined functions of methods whose type is a function or F# delegate type."
 3520,invalidXmlDocPosition,"XML comment is not placed on a valid language element."
 3521,tcInvalidMemberDeclNameMissingOrHasParen,"Invalid member declaration. The name of the member is missing or has parentheses."
+3522,tcAnonRecdDuplicateFieldId,"The field '%s' appears multiple times in this record expression or pattern."
```

Comment:
I think change to

```
3522,tcAnonRecdDuplicateFieldId,"The field '%s' appears multiple times in this anonymous record expression."
```
We don't actually support anonymous record patterns at all (at the moment, see RFC)

Reply:
Done